Dockerfile

```
@@ -0,0 +1,7 @@
+FROM node:15

```

Почему не 16я нода? Это сейчас актуальная LTS версия.

```suggestion
FROM alpine:3.16
```

Еще хорошо использовать `alpine` - это легковесные версии докер-образов, основанные на Alpine Linux

---

в учебнике советовали 15-ю, поэтому и поставил

---

если альпина, то ругается на отсутствие npm

---

указал 16-ю в общем